### ISSUE TITLE
Weird output of test_tensor_vec

### ISSUE THREAD

**Issue by atamazov** [ROCm/MIOpen#637]
I think this needs to be investigated.
<details><summary><b><i>Click to see screenshot</i></b></summary>

![image](https://user-images.githubusercontent.com/12697412/101146454-109b3480-362c-11eb-892c-59ad8537b5a1.png)
</details>

**Comment by ppanchad-amd** [comment]
@atamazov Does this still occur with ROCm 6.0.2?

**Comment by atamazov** [comment]
@ppanchad-amd Yes. Int8 values are still printed as characters. Please change urgency to https://github.com/ROCm/MIOpen/labels/urgency_low, and remove https://github.com/ROCm/MIOpen/labels/bug.

**Comment by JonathanLichtnerAMD** [comment]
Closing since MIOpen is being migrated to [ROCm/rocm-libraries](https://github.com/ROCm/rocm-libraries).  Please re-open there if you believe this is still an issue.
